packages/react-dev-utils/printBuildError.js

```diff
@@ -23,24 +22,22 @@ module.exports = function printBuildError(err) {
     message.indexOf('from UglifyJs') !== -1
   ) {
     try {
-      const matched = /Unexpected token:(.+)\[(.+):(.+),(.+)\]\[.+\]/.exec(
-        stack
-      );
+      const matched = /(.+)\[(.+):(.+),(.+)\]\[.+\]/.exec(stack);
       if (!matched) {
-        throw new Error(
-          "The regex pattern is not matched. Maybe UglifyJs changed it's message?"
-        );
+        throw new Error('Using errors for control flow is bad.');
```

Comment:
I'm short on time, but anyone who sees this can feel free to submit a PR removing the usage of errors for control flow.